Support/bone101/UI/bbui.js

```diff
@@ -909,6 +915,144 @@ var UI = (function() {
             return graph;
         })();
 
+        //graph global variables
```

Comment:
yes, they should be.

On 13 June 2016 at 15:42, Jason Kridner <notifications@github.com> wrote:

> In Support/bone101/UI/bbui.js
> <https://github.com/jadonk/bone101/pull/39#discussion_r66792824>:
>
> > @@ -909,6 +915,144 @@ var UI = (function() {
> >              return graph;
> >          })();
> >
> > +        //graph global variables
>
> If these are used for the graph alone, why aren't they part of the graph
> function?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jadonk/bone101/pull/39/files/3c6ed84f23c489652aebe3f4807f50d5c5db3333#r66792824>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG1g0pVyNFlIkBKdalZudF4d_zx-7xwHks5qLV43gaJpZM4IxAls>
> .
>